rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/380_sort_segments_on_timestamp.yml

```diff
@@ -131,6 +131,8 @@
       search:
         index: test_index3
         body:
+          # TODO: What is the default order for null
+          sort: '@timestamp'
```

Comment:
This test sometimes fail because of the result is returned in a different order. I cannot see how this could be related to CCS. It feels just because the null has an undefined order. But I'll ask the search team.

Reply:
Turns out this is a known issue and tracked at https://github.com/elastic/elasticsearch/issues/93572
I reverted changes to this file.